tools/azure-sdk-tools/packaging_tools/change_log.py

```diff
@@ -79,6 +79,16 @@ def operation(self, diff_entry):
             # Ignore change in metadata for now, they have no impact
             return
 
+        if remaining_path[0] == "parameters":
+            old_parameters_list = list(self._old_report["operations"][operation_name]["functions"][function_name]['parameters'])
+            new_parameters_list = list(self._new_report["operations"][operation_name]["functions"][function_name]['parameters'])
+            old_parameters = [param_name['name'] for param_name in old_parameters_list]
+            new_parameters = [param_name['name'] for param_name in new_parameters_list]
+            for removed_parameter in set(old_parameters) - set(new_parameters):
+                self.breaking_changes.append(_REMOVE_OPERATION_PARAM.format(operation_name, function_name, removed_parameter))
+            for added_parameter in set(new_parameters) - set(old_parameters):
+                self.breaking_changes.append(_ADD_OPERATION_PARAM.format(operation_name, function_name,added_parameter))
```

Comment:
We shall add a comment: 
we don't know the new parameter is optional or not. It may be breaking change. We could improve it later